main.tf

```diff
@@ -146,4 +147,38 @@ resource "aws_backup_selection" "default" {
       value = selection_tag.value["value"]
     }
   }
+
+  condition {
+    dynamic "string_equals" {
+      for_each = var.selection_conditions.string_equals
+      content {
+        key   = "aws:ResourceTag/${string_equals.value.key}"
```

Comment:
Is there other key prefixes that might fit here? Does it make more sense to have this be `string_equals.value.key` here and have consumers of this module pass `aws:ResourceTag/$THEIR_KEY` so that other consumers might be able to pass `aws:OtherTypeOfKey/$IDENTIFIER`? 

If the latter makes sense... can we switch this up? Can you document the common key prefixes that users of this module might use in the description of the `selection_conditions` variable?

Reply:
If that's the case, let's ship it 👍